nexus/src/app/mod.rs

```diff
@@ -245,6 +297,56 @@ impl Nexus {
         }
     }
 
+    // Called to hand off management of external servers to Nexus.
```

Comment:
Called by whom?

Reply:
Currently this happens to be called by `nexus::Server::start(...)` - I'm a little hesitant to reference that explicitly in a comment directly though, since that seems prone to bit-rot via refactor.